source/usage-examples/findOne.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+)
+
+func main() {
+
+	// Replace the uri string with your MongoDB deployment's connection string.
+	uri := "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority"
+	ctx := context.TODO()
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(uri))
+	if err != nil {
+		panic(err)
+	}
+	// Disconnect the client once the function returns.
```

Comment:
Remove this comment? It seems self evident in the code it describes.

Reply:
Thanks for the note. I was on the fence about this comment, but hearing  that you feel it is self evident I am alright removing it.

Fix: https://github.com/mongodb/docs-golang/pull/2/commits/2838ded48b9fbdd160dd030c30076ac0c3a7e7f9